### PR TITLE
Publish updates from next to main

### PR DESCRIPTION
**Pull request by CaroMac**

### CODE REVIEW

File: src/markdown-pages/docs/cli-command-reference/cli-command-reference-about.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+path: "/docs/cli-command-reference/cli-command-reference-about"
+title: "Galasa CLI commands"
+---
+
+Galasa CLI commands start with `galasactl`. Example commands are provided for running on Mac or Unix, and Windows Powershell. The Windows Powershell uses the backtick (`) for line continuation characters. If you are using Windows command-shell, the line continuation character is the caret (^). 
+
+You can view the Galasa CLI command syntax, including parameter descriptions, in the <a href=https://github.com/galasa-dev/cli/blob/main/docs/generated/galasactl.md target="_blank"> cli repository</a> in GitHub.
+
+
+## Getting help
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/cli-command-reference-about.md:11]: I still think it's weird talking about this before you get to the install instructions.

File: src/markdown-pages/docs/cli-command-reference/cli-command-reference.md [hunk 2]
```diff
@@ -6,37 +6,18 @@ title: "Getting started using the Galasa CLI"
...
-## About Galasa CLI commands
 
-Galasa CLI commands start with `galasactl`. Example commands are provided for running on Mac or Unix, and Windows Powershell. The Windows Powershell uses the backtick (`) for line continuation characters. If you are using Windows command-shell, the line continuation character is the caret (^). 
+## Prerequisites
 
-You can view the Galasa CLI command syntax, including parameter descriptions, in the <a href=https://github.com/galasa-dev/cli/blob/main/docs/generated/galasactl.md target="_blank"> cli repository</a> in GitHub.
+| Software |  Description  |
+| :---- | :-------- | 
+| Java JDK  | Required. Galasa tests and Managers are written in Java - you need to install a Java version 11 JDK or later to use it. _Note:_ We do not currently support Java 17 or later. |
+| Maven or Gradle  | You must install either Maven or Gradle in order to build Galasa projects. Galasa projects are hierarchical file structures that provide the ability to store and run Galasa tests. All Galasa versions are compatible with Gradle releases 6.8.x and later. |
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/cli-command-reference.md:17]: Not compatable with later versions of gradle. Thought we were fixing that.

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 3]
```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
...
+
+
+
+## Installing Maven
+
+Install Maven 3.8 or later. The following example uses Homebrew:
+
+1. Run the following command to install Maven:
+```
+brew update
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:72]: Brew is a mac-only thing. 
If we tell them how to install it, do we not have to tell windows users, and linux users how to install it ?

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 4]
```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
 
 You can now run the Galasa CLI too from any directory in your file system without having to specify the absolute path.
 
+## Installing Java 
+
+Install a Java version 11 JDK or later. We do not currently support Java 17 or later. The following example uses Homebrew to install Java version 11 on a MacOS.
+
+1. Install Java version 11 JDK using HomeBrew by running the following command in your terminal:
+```
+brew install openjdk@11
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:55]: Brew is a mac-only thing. What about windows or linux users ?

Do we have to tell them how to install it ? or should we do the work and tell them how to do it for all 3 environments ?

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 5]
```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
...
+     </profiles>
+</settings>
+```
+This information tells Maven where to find some custom Maven plug-in tools that are used by the build.
+
+
+## Installing Gradle
+
+1. Install <a href="https://gradle.org/install/" target="_blank"> Gradle</a> version 6.x. *Note:* Gradle version 7.x is not currently supported.
+2. Put Gradle on your PATH by adding the following information to your shell’s initialization file:
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:124]: I think this is fine. We are not telling people how to install gradle, just that it needs to be on the path.

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 6]
```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
...
+```
+export PATH="/opt/homebrew/opt/gradle@6/bin:$PATH"
+gradle --version | grep "Gradle" | cut -f2 -d' '
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:127]: The export puts it on your path.
The gradle --version line isn't needed.
Though you could say "to check that gradle is installed correctly, use gradle --version" or something ?

Comment (techcobweb) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:127]: Also, the export PATH=... is a unix/linux/mac thing. What's the windows equivalent ?

To be OS-agnostic, just saying add it to your PATH should be good enough, as Windows has a PATH environment variable also ?

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 7]
```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
...
+## About Galasa CLI commands
+
+Galasa CLI commands start with `galasactl`. Example commands are provided for running on Mac or Unix, and Windows Powershell. The Windows Powershell uses the backtick (`) for line continuation characters. If you are using Windows command-shell, the line continuation character is the caret (^). 
+
+You can view the Galasa CLI command syntax, including parameter descriptions, in the <a href=https://github.com/galasa-dev/cli/blob/main/docs/generated/galasactl.md target="_blank"> cli repository</a> in GitHub.
+
+
+## Known limitations
+
+Go programs can sometimes struggle to resolve DNS names, especially when a working over a virtual private network (VPN). In such situations, you might notice that a bootstrap file cannot be found with galasactl, but can be found by a desktop browser, or curl command. In such situations you can manually add the host detail to the `/etc/hosts` file, to avoid DNS being involved in the resolution mechanism.
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:139]: What is /etc/hosts on a windows machine ?

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 8]
```diff
@@ -3,15 +3,8 @@ path: "/docs/cli-command-reference/installing-cli-tool"
...
-| Maven or Gradle  | You must install either Maven or Gradle in order to build Galasa projects. Galasa projects are hierarchical file structures that provide the ability to store and run Galasa tests. All Galasa versions are compatible with Gradle releases 6.9.x. |
-| 3270 emulator | Optional. Although you do not need a 3270 emulator to run a Galasa test (even if it tests a 3270 application) you can use one to explore Galasa Simbank, a simulated version of an application that helps you get acquainted with Galasa before connecting to a real mainframe to run your own tests. There are many such emulators available but IBM's Personal Communications (PCOMM) is frequently used, as is IBM's Host on Demand software, which includes support for Windows, Linux and MacOS.| 
+This section provides details about how to download and install the Galasa command line interface tool (Galasa CLI) on your local machine and tells you a little bit about getting started with the Galasa CLI commands. 
 
 
 ## Downloading the Galasa CLI
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:10]: This should be preparing to install the galasa CLI ?

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 9]
```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
...
+
+Go programs can sometimes struggle to resolve DNS names, especially when a working over a virtual private network (VPN). In such situations, you might notice that a bootstrap file cannot be found with galasactl, but can be found by a desktop browser, or curl command. In such situations you can manually add the host detail to the `/etc/hosts` file, to avoid DNS being involved in the resolution mechanism.
+
+
+## Getting help
+
+Once you have installed the Galasa CLI, you can use the following command to get more information about the command and command options, including default values.
+
+```
+galasactl --help
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:147]: Isn't this duplicated in the file above ? Maybe it was copied and not removed from the original position ?

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 10]
```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
...
+
+
+## Installing Gradle
+
+1. Install <a href="https://gradle.org/install/" target="_blank"> Gradle</a> version 6.x. *Note:* Gradle version 7.x is not currently supported.
+2. Put Gradle on your PATH by adding the following information to your shell’s initialization file:
+```
+export PATH="/opt/homebrew/opt/gradle@6/bin:$PATH"
+gradle --version | grep "Gradle" | cut -f2 -d' '
+```
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:128]: I'm a bit lost. There's nothing here about actually installing the CLI ?

File: src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+
+After cloning the repository, complete the following steps to run the SimBankIVT test that is provided with Galasa. The following example uses SimBank OBR version `0.25.0` and Galasa uber OBR version `0.30.0`.
+
+You can find the version of the `dev.galasa.simbank.obr` that you are using by looking in the `pom.xml` file in the `dev.galasa.simbank.obr` folder. The `dev.galasa.uber.obr` is the OBR that contains all the bundles that are needed for Galasa to work (including the Managers, any required dependencies, the framework, etc). The version of the `dev.galasa.uber.obr` depends on which version of Galasa you have installed.
+
+
+1. Open a terminal, navigate to your `simplatform` directory and run the `./build-locally.sh` script to build the code in both the simbank-tests and the simplaform-application directories.
```
Comment (techcobweb) [inline src/markdown-pages/docs/cli-command-reference/running-simbank-tests-cli.md:57]: There is no windows build-locally or run-locally scripts.

File: src/markdown-pages/docs/first-steps/installing-offline.md [hunk 12]
```diff
@@ -35,6 +35,31 @@ Depending on how you use Galasa, there are several software prerequisites, some
 | 3270 emulator | Optional. Although you do not need a 3270 emulator to run a Galasa test (even if it tests a 3270 application) you can use one to explore Galasa Simbank, a simulated version of an application that helps you get acquainted with Galasa before connecting to a real mainframe to run your own tests. There are many such emulators available but IBM's Personal Communications (PCOMM) is frequently used, as is IBM's Host on Demand software, which includes support for Windows, Linux and MacOS.| 
 
 
+## Installing Java 
+
+Install a Java version 11 JDK or later. We do not currently support Java 17 or later. The following example uses Homebrew to install Java version 11 on a MacOS.
+
+1. Install Java version 11 JDK using HomeBrew by running the following command in your terminal:
+```
+brew install openjdk@11
```
Comment (techcobweb) [inline src/markdown-pages/docs/first-steps/installing-offline.md:44]: duplicate information with above file ? I've seen this before in this review...

File: src/markdown-pages/docs/cli-command-reference/cli-command-reference-about.md (hunk 1 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/cli-command-reference-about.md:11]: have switched around

File: src/markdown-pages/docs/cli-command-reference/cli-command-reference.md (hunk 2 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/cli-command-reference.md]: yes, that was fixed. think a problem with merge. now resolved.

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md [hunk 13]
```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
...
+                    <id>galasa.repo</id>    
+                    <url>https://development.galasa.dev/main/maven-repo/obr</url> 
+                </pluginRepository>
+                -->
+             </pluginRepositories>
+         </profile>
+     </profiles>
+</settings>
+```
+This information tells Maven where to find some custom Maven plug-in tools that are used by the build.
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md]: added in a check for .m2 repo config checking as per discussion

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md (hunk 4 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md]: see previous comment

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md (hunk 5 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md]: ok

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md (hunk 6 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md]: removed this

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md (hunk 7 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md]: it is here c:\Windows\System32\Drivers\etc\hosts, so will update to be \etc\hosts

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md (hunk 8 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md:10]: have restructured

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md (hunk 9 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md]: removed dupe thank you

File: src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md (hunk 10 above)
Comment (CaroMac) [reply src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md]: removed this

File: src/markdown-pages/docs/first-steps/installing-offline.md (hunk 12 above)
Comment (CaroMac) [reply src/markdown-pages/docs/first-steps/installing-offline.md]: took this out in the end - see earlier comment

File: src/markdown-pages/docs/first-steps/installing-online.md [hunk 14]
```diff
@@ -23,6 +23,31 @@ Depending on how you plan to use Galasa, there are several software prerequisite
 | 3270 emulator | Optional. Although you do not need a 3270 emulator to run a Galasa test (even if it tests a 3270 application) you can use one to explore Galasa Simbank, a simulated version of an application that helps you get acquainted with Galasa before connecting to a real mainframe to run your own tests. There are many such emulators available but IBM's Personal Communications (PCOMM) is frequently used, as is IBM's Host on Demand software, which includes support for Windows, Linux and MacOS.| 
 
 
+## Installing Java 
+
+Install a Java version 11 JDK or later. We do not currently support Java 17 or later. The following example uses Homebrew to install Java version 11 on a MacOS.
+
+1. Install Java version 11 JDK using HomeBrew by running the following command in your terminal:
+```
+brew install openjdk@11
```
[the remark this answers is not in the document]
Comment (CaroMac) [reply src/markdown-pages/docs/first-steps/installing-online.md]: see previous comment

**Review by techcobweb — APPROVED** [review]

**Pull request merged**